src/IO_Strings.f90

```diff
@@ -1722,4 +1724,48 @@ FUNCTION getRowStr(maxcolsize) RESULT(rowstr)
   ENDDO
 ENDFUNCTION getRowStr
 !
+!------------------------------------------------------------------------------
+!> @brief Removes duplicate entries from a list of strings
+!> @param list the list from which to remove dupliates
+!>
+!> The list is sorted before removing duplicates, so the returned list will
+!> be sorted compared to the initial list.
+!>
+SUBROUTINE removeDuplicates(list)
+  TYPE(StringType),ALLOCATABLE,INTENT(INOUT) :: list(:)
+  !
+  INTEGER(SIK) :: nUnique,iList
+  TYPE(StringType),ALLOCATABLE :: old_list(:)
+
+  IF(.NOT.ALLOCATED(list)) THEN
+    ALLOCATE(list(0))
+  ENDIF
```

Comment:
would we prefer a DBC for allocation instead of just having it return?

Reply:
I lean toward returning (not strongly), though after considering it couldn't it return if size is < 2. Since a size 1 list can't have a duplicate.

Reply:
I dunno.  I don't really have a preference.  I just went for robustness on this one